resources/assets/lib/chat/message-group.tsx

```diff
@@ -30,9 +29,9 @@ export default class MessageGroup extends React.Component<Props, any> {
     return (
       <div className={className}>
         <div className='chat-message-group__sender'>
-          <a className='js-usercard' data-tooltip-position='top center' data-user-id={sender.id} href={route('users.show', { user: sender.id })}>
```

Comment:
apparently the `tooltip-position` is supposed to work except it's not anymore when it's converted to typescript. Should probably at least keep the parameter intact for when it's supported again.

also `top center` is not the default for user card. It's hardcoded to `right center` at the moment which is the original default.